### ISSUE TITLE
[Feature] need to output prompt logits

### ISSUE THREAD

**Issue by anaivebird** [InternLM/lmdeploy#2614]
### Motivation

How to output logits of prompt input, and it will be needed as some evaluation tasks.

**Comment by lvhan028** [comment]
In the [pipeline guide](https://lmdeploy.readthedocs.io/en/latest/llm/pipeline.html), there is an example you may follow:
```python
from transformers import AutoTokenizer
from lmdeploy import pipeline
model_repoid_or_path='internlm/internlm2_5-7b-chat'
pipe = pipeline(model_repoid_or_path)
tokenizer = AutoTokenizer.from_pretrained(model_repoid_or_path, trust_remote_code=True)

# logits
messages = [
   {"role": "user", "content": "Hello, how are you?"},
]
input_ids = tokenizer.apply_chat_template(messages)
logits = pipe.get_logits(input_ids)

# ppl
ppl = pipe.get_ppl(input_ids)
```

**Comment by anaivebird** [comment]
Thanks, how could I do it in openai compatible server since we need call it with api.

**Comment by lvhan028** [comment]
api_server didn't release get_logits API
